skopt/space.py

```diff
@@ -10,6 +10,11 @@
 from sklearn.utils.fixes import sp_version
 
 
+class _Ellipsis:
```

Comment:
Surely just printing out `...` is easier than the burden of maintaining a separate class? :p